app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb

```diff
@@ -77,11 +77,13 @@ def get_volumes
       def get_snapshots
         snapshots = @connection.snapshots.all
         process_collection(snapshots, :cloud_volume_snapshots) { |snapshot| parse_snapshot(snapshot) }
+        # Also pass the snapshots as templates
+        process_collection(snapshots, :vms) { |snapshot| parse_storage_as_template(snapshot) }
```

Comment:
@agrare For VMware provisioning we allow you to provision a linked-clone which requires you to select a snapshot.  Does that align better with what you are trying to do here?

![image](https://cloud.githubusercontent.com/assets/2591569/14568787/178c86d2-0309-11e6-88ca-ec2020703551.png)